web/versioned_docs/version-0.13.0/auth/social-auth/discord.md

```diff
@@ -0,0 +1,549 @@
+---
```

Comment:
The docs won't build for me:
```
[ERROR] Error: Invalid sidebar file at "sidebars.js".
These sidebar document ids do not exist:
- auth/social-auth/discord

Available document ids are:
- advanced/accessing-app-config
- advanced/apis
- advanced/deployment/cli
- advanced/deployment/manually
- advanced/deployment/overview
- advanced/email/email
- advanced/jobs
- advanced/links
- advanced/middleware-config
- advanced/web-sockets
- auth/auth-hooks
- auth/email
- auth/entities/entities
- auth/overview
- auth/social-auth/github
- auth/social-auth/google
- auth/social-auth/keycloak
- auth/social-auth/overview
- auth/ui
- auth/username-and-pass
- contact
- contributing
- data-model/backends
- data-model/crud
- data-model/entities
- data-model/operations/actions
- data-model/operations/overview
- data-model/operations/queries
- general/cli
- general/language
- introduction/editor-setup
- introduction/introduction
- introduction/quick-start
- migrate-from-0-11-to-0-12
- migrate-from-0-12-to-0-13
- migrate-from-0-13-to-0-14
- project/client-config
- project/css-frameworks
- project/custom-vite-config
- project/customizing-app
- project/dependencies
- project/env-vars
- project/server-config
- project/starter-templates
- project/static-assets
- project/testing
- telemetry
- tutorial/actions
- tutorial/auth
- tutorial/create
- tutorial/entities
- tutorial/pages
- tutorial/project-structure
- tutorial/queries
- vision
- wasp-ai/creating-new-app
- wasp-ai/developing-existing-app
- writingguide
```

This is due to you putting the Discord docs _only_ in the 0.13.0 versioned docs. You need to put them in the `docs/` folder which are the _latest_ docs for the docs to work properly.

Reply:
Did you try running the docs with `npm run start` in the `web` folder to see them working?

Reply:
I did not, wasn't sure how to do that - will update and try this